.changeset/string-comparators-changeset.md

```diff
@@ -0,0 +1,7 @@
+---
+"graphql-manual": patch
+"@neo4j/graphql": patch
```

Comment:
I think this should be a minor release, as it is adding some features

Reply:
I agree!